tools/explorer/lib/tests/ontology_wrapper_test.py

```diff
@@ -59,6 +60,69 @@ def testGetRequiredFieldsForTypeName(self):
 
     self.assertEqual(function_output, expected_output)
 
+  def testGetEntityTypesFromFields(self):
+    input_field_list = [
+        StandardField('', 'exhaust_air_damper_command'),
+        StandardField('', 'exhaust_air_damper_status'),
+        StandardField('', 'manufacturer_label'),
+        StandardField('', 'model_label')
+    ]
+    etu = self.universe.entity_type_universe
+    expected_output = [
+        Match(input_field_list, etu.GetEntityType('HVAC', 'DMP_EDM'), 100),
+        Match(input_field_list, etu.GetEntityType('HVAC', 'SDC_EXT'), 25),
+        Match(input_field_list, etu.GetEntityType('HVAC', 'CHWS_WDT'), 0)
+    ]
+
+    function_output = self.ontology.GetEntityTypesFromFields(input_field_list)
+
+    for i in range(len(expected_output)):
```

Comment:
is the for loop needed here?

Reply:
when I don't compare the elements one by one in the loop, the test does not pass. I think it's because wrote __eq__() for the match data type. comparing the lists doesn't seem to work.